packages/protocol/src/data/part-data.ts

```diff
@@ -52,3 +52,7 @@ export interface SignalCatchData {
   signalCode: string;
   attachToBusinessCase: boolean;
 }
+
+export interface HeaderData {
+  headers: MailHeaders;
```

Comment:
I did not see that this already existed 😮
otherwise ther rest did almost the same 😃